packages/edit-site/src/components/site-editor-routes/README.md

```diff
@@ -0,0 +1,20 @@
+# Site Editor Routes
+
+## Areas
+
+When `canvasMode` is not `edit`, the areas avaliable to use are:
```

Comment:
Oh I see, for me I see things differently :) 

I see things like that:

the available areas are always "all of them" but when the canvas mode is "edit", the "preview" area is "focused/zoomed".

Reply:
Actually, I think we should document that the areas are different based on viewport, not based on canvasMode.

In mobile: the "mobile" area is used or otherwise "preview" (fallback)
In desktop: All the areas are used.

or something like that.